src/app/extensions.cpp

```diff
@@ -817,33 +817,33 @@ Extensions::Extensions()
   // Load extensions from data/ directory on all possible locations
   // (installed folder and user folder)
   while (rf.next()) {
-    auto extensionsDir = rf.filename();
+    const auto& extensionsDir = rf.filename();
 
-    if (base::is_directory(extensionsDir)) {
-      for (auto fn : base::list_files(extensionsDir)) {
-        const auto dir = base::join_path(extensionsDir, fn);
-        if (!base::is_directory(dir))
-          continue;
+    if (!base::is_directory(extensionsDir))
+      continue;
 
-        const bool isBuiltinExtension =
-          (m_userExtensionsPath != base::get_file_path(dir));
+    for (auto& fn : base::list_files(extensionsDir)) {
+      const auto dir = base::join_path(extensionsDir, fn);
+      if (!base::is_directory(dir))
+        continue;
 
-        auto fullFn = base::join_path(dir, kPackageJson);
-        fullFn = base::normalize_path(fullFn);
+      const bool isBuiltinExtension = m_userExtensionsPath != base::get_file_path(dir);
```

Comment:
Keep the parenthesis and the line break as it was: https://github.com/aseprite/aseprite/blob/c8f018f2f1a7f050ca79aa6b766bb4d4fb86168f/docs/CODING_STYLE.md?plain=1#L32-L34

```c++
      const bool isBuiltinExtension =
        (m_userExtensionsPath != base::get_file_path(dir));
```